### PR TITLE
PIM-5051 : Bulk remove throw an exception on versionning

### PR DESCRIPTION
**Pull request by jmleroux**
| Q                 | A
| ----------------- | ---
| Specs             |updated
| Behats            |no new feature
| Blue CI           |running
| Changelog updated |yes
| Review and 2 GTM  |OK

### CODE REVIEW

File: src/Pim/Bundle/VersioningBundle/EventSubscriber/AddRemoveVersionSubscriber.php [hunk 1]
```diff
@@ -97,6 +97,7 @@ public function postRemove(RemoveEvent $event)
             ->setSnapshot(null !== $previousVersion ? $previousVersion->getSnapshot(): [])
             ->setChangeset([]);
 
-        $this->versionSaver->save($version);
+        $options = $event->getArguments();
+        $this->versionSaver->save($version, $options);
```
Comment (grena) [inline src/Pim/Bundle/VersioningBundle/EventSubscriber/AddRemoveVersionSubscriber.php]: Will it fail if the event has no argument?

**Comment by grena** [comment]
Question, but GTM

**Comment by solivier** [comment]
GTM

File: src/Pim/Bundle/VersioningBundle/EventSubscriber/AddRemoveVersionSubscriber.php (hunk 1 above)
Comment (fitn) [inline src/Pim/Bundle/VersioningBundle/EventSubscriber/AddRemoveVersionSubscriber.php]: As the RemoveEvent takes an empty array as default parameter, it won't fail ;)

**Pull request merged**